src/main/kotlin/org/radarbase/push/integration/garmin/service/GarminHealthApiService.kt

```diff
@@ -107,7 +110,9 @@ class GarminHealthApiService(
 
     @Throws(IOException::class, BadRequestException::class)
     fun processManualActivities(tree: JsonNode, user: User): Response {
-        return this.processActivities(tree, user)
+        val records = manualActivitiesConverter.validateAndConvert(tree, user)
+        producerPool.produce(manualActivitiesConverter.topic, records)
+        return Response.status(OK).build()
```

Comment:
Alternatives:
- Don't return a OK response, it is the default response by Jersey so it will be redundant.
- Use the shortcut `Response.ok().build()`.

Reply:
The second alternative looks good to me. It offers flexibility to be changed if required in the future.